hw/ip/pinmux/doc/padctrl.md

```diff
@@ -155,20 +159,23 @@ One possible future enhancement is to individually lock each register instead of
 Note that the register description given in the next section is an example that has been generated with the default parameterization, and the layout may change once reparameterized.
 The following pad attributes are supported by this register layout by default:
 
-ATTR Bits | Description                                 | Access
-----------|---------------------------------------------|---------
-0         | Input/output inversion                      | RW
-1         | Open drain enable                           | RW
-2         | Pull enable                                 | WARL
-3         | Pull select (0: down, 1: up)                | WARL
-4         | Keeper enable                               | WARL
-5         | Schmitt trigger enable                      | WARL
-6         | Slew rate (0: slow, 1: fast)                | WARL
-8:7       | Drive strengh (00: weakest, 11: strongest)  | WARL
-9         | Reserved                                    | WARL
+ATTR Bits | Description                                   | Access
+----------|-----------------------------------------------|---------
+0         | Input/output inversion                        | RW
+1         | Open drain enable                             | RW
+2         | Pull enable                                   | WARL
+3         | Pull select (0: down, 1: up)                  | WARL
+4         | Keeper enable                                 | WARL
+5         | Schmitt trigger enable                        | WARL
+6         | Open drain enable enable                      | WARL
+8:7       | Slew rate (0x0: slowest, 0x3: fastest)        | WARL
+12:9      | Drive strength (0x0: weakest, 0xf: strongest) | WARL
```

Comment:
@silvestrst FYI: this is the attribute update I have been talking about.

Reply:
@msfschaffner thanks!